bdd_tests/features/steps/common.py

```diff
@@ -0,0 +1,40 @@
+# Copyright © 2021 Pavel Tisnovsky, Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from shutil import which
+import psycopg2
+from psycopg2.errors import UndefinedTable
+
+
+from behave import given, then, when
+
+
+@given(u"the system is in default state")
+def system_in_default_state(context):
+    """Check the default system state."""
+    pass
+
+
+@when(u"I look for executable file {filename}")
+def look_for_executable_file(context, filename):
```

Comment:
This function actually looks for the executable in PATH, not whether it exists or not. I would change the logic with `file_was_found` function or use `os.path.isfile('file')`.

Reply:
I'd rather change the name =)

But like Pavel mentioned, this part is copy-pasted from the ccx-notification-writer BDD testcases. I'll leave it as is for now, and take it into account when we do some unification work.